README.md

```diff
@@ -189,11 +189,14 @@ Currently we only support one material type per mesh (see [known limitations](#k
 
 ## Release Notes
 
-### v2.2.0 (2023-01-XX)
-Required Cityengine version: 2022.1 or older.
+### v2.2.0 (2023-06-XX)
+Required Cityengine version: 2023.0 or older.
+
+#### Added:
+* Support for Maya 2023. (nolonger support Maya 2019)
```

Comment:
Please add a "Removed" section and put Maya 2019 there: "Removed support for Maya 2019"

Reply:
added "Removed" section